### PR TITLE
#29 fixing getAutorizablesWithRetry implementation

### PR DESCRIPTION
**Pull request by tyge68**
Refactoring getAutorizablesWithRetry to avoid re-calculation of getAuthorizables  which fails randomly.
Removing unused method
Adding log for created user tracking.

### CODE REVIEW

**Comment by tyge68** [comment]
@volteanu  can you check this PR ? I guess the coverage drop is cosmetic here.

File: src/main/java/com/adobe/cq/testing/client/security/AbstractAuthorizable.java [hunk 1]
```diff
@@ -236,17 +236,20 @@ private String getAuthorizableType(String authorizableId) throws ClientException
     }
 
     private JsonNode getAutorizablesWithRetry(final String query) throws ClientException, InterruptedException {
+        final JsonNode[] jsonNode = new JsonNode[1];
```
Comment (volteanu) [inline src/main/java/com/adobe/cq/testing/client/security/AbstractAuthorizable.java]: Not sure I understand the role of the array. FWIU, you want to save the result of `getAuthorizables()` and return it in the end. Wouldn't a simple reference be enough for that?

Comment (tyge68) [reply src/main/java/com/adobe/cq/testing/client/security/AbstractAuthorizable.java]: @volteanu  element accessed in inner class need to be final or effectively final, thus the only way is to store it in an array object that is final and then read it after that.

**Comment by volteanu** [comment]
thanks a lot @tyge68 for this PR, much appreciated! I've made a slight change to the polling and merged your commit. Will do a release asap.

**Pull request merged**